composer/packages/doc-search/src/components/list.tsx

```diff
@@ -0,0 +1,332 @@
+            }
+        }
+        // Set the filtered state based on what our rules added to newList
+        this.setState({
+            filteredModules: newModuleList,
+            filteredFunctions: newFunctionsList,
+            filteredObjects: newObjectsList,
+            filteredRecords: newRecordsList,
+            filteredConstants: newConstantsList,
+            filteredErrors: newErrorsList,
+            filteredTypes: newTypesList
+        });
+    }
+
+    render() {
+        return (
+            <div>
+                <div className="ui fluid icon input search-bar">
+                    <input type="text" id="searchBox" onKeyUp={this.handleChange} placeholder="Search..." />
+                    <i className="search icon"></i>
+                </div>
+
+                {/* <input type="text" id="searchBox" className="input" onKeyUp={this.handleChange} placeholder="Search..." defaultValue={this.state.searchText} /> */}
+                {/* <button className="button is-info" onClick={this.handleChange}>Search</button> */}
+
+                {this.state.searchText &&
+                    <div className="search-list">
+                        <h1>Search results for '{this.state.searchText}'</h1>
+                        {this.state.filteredModules.length > 0 &&
+                            <div>
+                                <h3>Modules: {this.state.filteredModules.length}</h3>
+                                <table>
+                                    <tbody>
+                                        {this.state.filteredModules.map(item => (
+                                            <tr>
+                                                <td className="search-title" id={item.id} title={item.id}>
+                                                    <a href={rootPath + item.id + "/index.html"} className="objects">{item.id}</a></td>
+                                                <td className="search-desc"><span dangerouslySetInnerHTML={{ __html: item.searchString }} /></td>
+                                            </tr>
+                                        ))}
+                                    </tbody>
+                                </table>
+                            </div>
+                        }
+
+                        {this.state.filteredObjects.length > 0 &&
+                            <div>
+                                <h3>Objects: {this.state.filteredObjects.length}</h3>
+                                <table>
+                                    <tbody>
+                                        {this.state.filteredObjects.map(item => (
+                                            <tr>
+                                                <td className="search-title" id={item.id} title={item.id}>
+                                                    <a href={rootPath + item.moduleId + "/objects/" + item.id + ".html"} className="objects">{item.moduleId + ": " + item.id}</a></td>
```

Comment:
This line has more than 120 characters. Shall we find all such lines and break those into not more than 120 character lines?

Reply:
Added changes